homeassistant/components/generic/strings.json

```
@@ -55,6 +55,7 @@
           "authentication": "[%key:component::generic::config::step::user::data::authentication%]",
           "limit_refetch_to_url_change": "[%key:component::generic::config::step::user::data::limit_refetch_to_url_change%]",
           "password": "[%key:common::config_flow::data::password%]",
+          "use_wallclock_as_timestamps": "Advanced: Use wallclock as timestamps (usually not required)",
```

```suggestion
          "use_wallclock_as_timestamps": "Use wallclock as timestamps",
```

And then let's add that it's for troubleshooting in [`data_description` field](https://github.com/home-assistant/core/blob/dev/homeassistant/components/denonavr/strings.json#L9).